### PR TITLE
Add generated files and frontmatter from csv

### PR DESCRIPTION
**Pull request by rrcobb**

### CODE REVIEW

File: T002-Working-with-Variables/p006-variables-check-in.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: 'Variables Check-in'
+slug: 'variables-check-in'
+contentType: ''
```
Comment (rrcobb) [inline T002-Working-with-Variables/p006-variables-check-in.mdx:4]: 👀

File: T012-Using-Your-Python-Skills/p007-what-s-next.mdx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: 'What's Next'
```
Comment (dionlarson) [inline T012-Using-Your-Python-Skills/p007-what-s-next.mdx:2]:
```suggestion
title: "What's Next"
```

**Review by dionlarson — CHANGES_REQUESTED** [review]
@rrcobb Not sure if you want to use use `"` by default or escape `'` in the generation script.

**Comment by rrcobb** [comment]
> @rrcobb Not sure if you want to use use `"` by default or escape `'` in the generation script.

Added a naive switch in the script to double quote if the title has a `'`. There's probably lots of other fragility to the script...

**Pull request merged**